src/eval-frame/actions/actions.js

```diff
@@ -275,6 +277,13 @@ function evaluateCSSCell(cell) {
   }
 }
 
+function sendKernelStateAndEvaluate(pr, dispatch, cell, evaluator) {
```

Comment:
what is "pr"? i'm sure i'm missing something obvious, but literally can't figure out what it's short for, which i guess is a sign we should use a more descriptive name here

Reply:
oh it's "promise". i know it should be obvious, but i missed it for a long time, so above still stands... let's help the slow and sleep deprived :-)